dkobo/koboform/views/survey_draft_views.py

```diff
@@ -109,18 +117,31 @@ def survey_draft_detail(request, pk, format=None):
     "application/octet-stream",
 ]
 
-@login_required
 def bulk_delete_questions(request):
     question_ids = json.loads(request.body)
     SurveyDraft.objects.filter(user=request.user).filter(id__in=question_ids).delete()
     return HttpResponse('')
 
-@login_required
 def import_survey_draft(request):
     """
     Imports an XLS or CSV file into the user's SurveyDraft list.
     Returns an error in JSON if the survey was not valid.
     """
+    myw_kobo_user_cookie = request.COOKIES.get('myw_kobo_user')
+    email = ''
+    if not myw_kobo_user_cookie:
+        return HttpResponse('Json Web Token NOT set.')
+
+    token_payload = jwt.decode(myw_kobo_user_cookie,
```

Comment:
this is repeated, if you abstract above with this piece as a helper you can call that helper instead of repeating here